amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -4018,6 +4230,139 @@ definitions:
     properties:
       reason:
         type: string
+  RetentionNodeBodyCreate:
+    type: object
+    properties:
+      authority:
+        type: string
+      instructions:
+        type: string
+      isRecordLevel:
+        type: boolean
+        default: false
+  RetentionScheduleResponse:
+    type: object
+    properties:
+      id:
+        type: string
+      parentId:
+        type: string
+      authority:
+        type: string
+      instructions:
+        type: string
+      isRecordLevel:
+        type: boolean
+      unpublishedUpdates:
+        type: boolean
+  RetentionScheduleResponseList:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/FullRetentionScheduleResponse'
+  FullRetentionScheduleResponse:
+    type: object
+    properties:
+      id:
+        type: string
+      parentId:
+        type: string
+      authority:
+        type: string
+      instructions:
+        type: string
+      isRecordLevel:
+        type: boolean
+      unpublishedUpdates:
+        type: boolean
+      actions:
+        type: array
+        items:
+          $ref: '#/definitions/Actions'
+      events:
+        type: array
+        items:
+          type: string
+  Actions:
+    type: object
+    properties:
+      id:
+        type: string
+      name:
+        type: string
+      periodAmount:
+        type: integer
+      period:
+        type: string
+      periodProperty:
+        type: string
+      combineDispositionStepConditions:
+        type: boolean
+        default: false
+      eligibleOnFirstCompleteEvent:
+        type: boolean
+        default: true
+      description:
+        type: string
+      destroyWithGhosting:
+        type: boolean
+      location:
+        type: string
+      index:
+        type: integer
+  RetentionStepNodeBodyCreate:
+    type: object
+    properties:
+      name:
+        type: string
+      periodAmount:
+        type: integer
+      period:
+        type: string
+      periodProperty:
+        type: string
+      combineDispositionStepConditions:
+        type: boolean
+      events:
+        type: array
+        items:
+          type: string
+      eligibleOnFirstCompleteEvent:
+        type: boolean
+      description:
+        type: string
+      destroyWithGhosting:
```

Comment:
I think it makes more sense to have two different destroy step types: "destroy-node" and "destroy-content"
In the application we can map "destroy-content" to "destroy" with `destroyWithGhosting` set to true.

Reply:
added two different destroy step type as per the comment.